scraper.js

```diff
@@ -30,14 +30,15 @@ optStrings = {
 currentState = {
     preloadCount: 0,
     finished: false,
+    limit: 100
```

Comment:
Let's default the limit to 0.
By defaulting to 100 we are setting the limit without the user's knowledge therefore hiding program logic.

Reply:
Oh, didn't see that you default to 0 on the flag config. This should still be changed to keep the consistency between the scripts.